code/src/bip/datamodel/constants.py

```diff
@@ -0,0 +1,86 @@
+from django.utils.translation import ugettext_lazy as _
+
+from djchoices import ChoiceItem, DjangoChoices
+
+
+class PostadresType(DjangoChoices):
+    antwoordnummer = ChoiceItem(
+        'Antwoordnummer',
+        label=_('Antwoordnummer')
+    )
+    postbusnummer = ChoiceItem(
+        'Postbusnummer',
+        label=_('Postbusnummer')
+    )
+
+
+class GeslachtsAanduiding(DjangoChoices):
+    man = ChoiceItem(
+        'Man',
+        label=_('Man')
+    )
+    vrouw = ChoiceItem(
+        'Vrouw',
+        label=_('Vrouw')
+    )
+    onbekend = ChoiceItem(
+        'Onbekend',
+        label=_('Onbekend')
+    )
+
+class AdellijkeTitel(DjangoChoices):
+    baron = ChoiceItem('B', 'Baron')
+    barones = ChoiceItem('B', 'Barones')
```

Comment:
zelfde waarde voor ander label? Dan kan je niet meer uit waarde het label afleiden...

Reply:
Volgens het RSGB is dat correct. De juiste afleiding tot de naam valt te halen uit het geslacht (ik heb het niet bedacht)

In deze nieuwe API zou ik dat anders doen maar ik zie net dat deze uit de OAS is gehaald in zijn geheel, dus maar verwijdert.